dashboards/kubelet.libsonnet

```diff
@@ -17,7 +17,7 @@ local singlestat = grafana.singlestat;
           span=2,
           valueName='min',
         )
-        .addTarget(prometheus.target('sum(up{%(clusterLabel)s="$cluster", %(kubeletSelector)s})' % $._config));
```

Comment:
@paulfantom @brancz  this change breaks kube-prometheus.
https://github.com/prometheus-operator/kube-prometheus/blob/07136d1d6e0995908f3b418e4d8337cf769e1006/jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet#L13 already has `kubeletSelector: 'job="kubelet", metrics_path="/metrics"'`  defined.   Also, above introduces a syntactical error in prom query expression.